YAXLib/Deserialization.cs

```diff
@@ -808,6 +808,27 @@ private bool TrySetValueForEmptyElement(object obj, MemberWrapper member, Type m
         if (realType != null) memberType = realType;
     }
 
+    private IList? TryGetCollectionItemList(Type collectionItemType)
+    {
+        if (collectionItemType == typeof(object))
+        {
+            return new List<object>();
+        }
+        else
+        {
+            try
+            {
+                var listType = typeof(List<>).MakeGenericType(collectionItemType);
+                var result = (IList) Activator.CreateInstance(listType)!;
+                return result;
+            }
+            catch (Exception)
+            {
+                return null;
+            }
+        }
```

Comment:
If something unforeseeable happens, it's better to throw than swallow the `Exception`